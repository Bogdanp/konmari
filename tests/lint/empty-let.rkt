#lang racket/base

(let ([x 1]))

(let loop ())

(let loop ()
  (loop))
