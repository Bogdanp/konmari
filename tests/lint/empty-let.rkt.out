/empty-let.rkt:3:1:error:let forms must contain at least one body expression
/empty-let.rkt:3:8:warning:identifier 'x' is never used
/empty-let.rkt:5:1:error:let forms must contain at least one body expression
/empty-let.rkt:5:6:warning:identifier 'loop' is never used
